Tests/test_map_ui.gd

```diff
@@ -1,22 +1,22 @@
-extends GutTest
+extends TestMapBase
 
 var map_ui_scene: PackedScene = load("res://#Scenes/MapUI.tscn")
 var map_ui: MapUI
 var test_map_array: Array[int] = [1, 3, 5, 3, 1]
```

Comment:
since you extend from `TestMapBase`, you don't need to declare most of those variables anymore (check `TestMapBase`)